README.md

```diff
@@ -2,55 +2,55 @@
 
 [![Build Status](https://travis-ci.org/vulcanize/VulcanizeDB.svg?branch=master)](https://travis-ci.org/vulcanize/VulcanizeDB)
 
-### Dependencies
-
+## Dependencies
  - Go 1.9+
- - Postgres 10 
+ - Postgres 10
  - Ethereum Node
    - [Go Ethereum](https://ethereum.github.io/go-ethereum/downloads/) (1.8+)
    - [Parity 1.8.11+](https://github.com/paritytech/parity/releases)
-   
-### Installation 
-`go get github.com/vulcanize/vulcanizedb`
 
-### Setting up the Databases
+## Installation
+`go get github.com/vulcanize/vulcanizedb`
 
+## Setting up the Database
 1. Install Postgres
 1. Create a superuser for yourself and make sure `psql --list` works without prompting for a password.
-1. `createdb vulcanize_private`
+1. `createdb vulcanize_public`
 1. `cd $GOPATH/src/github.com/vulcanize/vulcanizedb`
-1. Import the schema: `psql vulcanize_private < db/schema.sql`
+1. Import the schema: `psql vulcanize_public < db/schema.sql`
 
    or run the migrations: `make migrate HOST_NAME=localhost NAME=vulcanize_public PORT=5432`
     * See below for configuring additional environments
-    
-### IPC File Paths
-
-The default location for Ethereum is:
- - `$HOME/Library/Ethereum` for Mac
- - `$HOME/.ethereum` for Ubuntu
- - `$GOPATH/src/gihub.com/vulcanize/vulcanizedb/test_data_dir/geth.ipc` for private node.
 
-**Note the location of the ipc file is printed to the console when you start geth. It is needed to for configuration**
+## Configuration
+- To use a local Ethereum node, copy `environments/public.toml.example` to
+  `environments/public.toml` and update the `ipcPath` to the local node's IPC filepath:
+  - when using geth:
+    - The IPC file is called `geth.ipc`.
+    - The geth IPC file path is printed to the console when you start geth.
+    - The default location is:
+      - Mac: `$HOME/Library/Ethereum`
+      - Linux: `$HOME/.ethereum`
+
+  - when using parity:
+    - The IPC file is called `jsonrpc.ipc`.
+    - The default location is:
+      - Mac: `$HOME/Library/Application\ Support/io.parity.ethereum/`
+      - Linux: `$HOME/.local/share/io.parity.ethereum/`
+
+- `environments/infura.toml` can be used to run commands with a full
```

Comment:
I might rephrase as "run commands against infura, if local node is unavailable." You could technically point any of the config files at a full archive node if you had your local node configured in archive mode

Reply:
Good call, updated in 194d81f